app/src/globalConfig.ts

```diff
@@ -110,8 +112,12 @@ export const initLanguage = () => {
                     navigator.languages.includes(l.languageCode),
                 )?._id;
 
-                // If a browser preferred language exists, set it
-                if (browserPreferredLanguageId) {
+                // If a browser preferred language exists, set it if the ordering has not
+                // already been changed by the user.
+                if (
+                    browserPreferredLanguageId &&
+                    appLanguageIdsAsRef.value !== appLanguageIdsCopy.value
```

Comment:
Would this also work?
```suggestion
                    browserPreferredLanguageId && appLanguageIdsAsRef.value.length == 0
```

Reply:
Suggestion added